### PR TITLE
 session3/chapter3/slow-query-table.md 

### PR DESCRIPTION
**Pull request by aylen**
update slow query table

### CODE REVIEW

File: session3/chapter3/slow-query-table.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+where is_internal = false  -- 排除 TiDB 内部的慢查询 SQL
+
+order by query_time desc
+
+limit 2;
+```
+输出样例：
+```
++--------------+------------------------------------------------------------------+
+
```
Comment (c4pt0r) [inline session3/chapter3/slow-query-table.md]: 空行请去掉

File: session3/chapter3/slow-query-table.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+order by query_time desc
+
+limit 2;
+```
+输出样例：
+```
++-------------+------------------------------------------------------------------+----------------+
+
+| Query_time  | query                                                            | user           |
+
```
Comment (c4pt0r) [inline session3/chapter3/slow-query-table.md]: same as above

Comment (aylen) [reply session3/chapter3/slow-query-table.md]: 已去除

**Review by crazycs520 — COMMENTED** [review]
LGTM

**Review by winkyao — APPROVED** [review]
LGTM

**Pull request merged**